src/error.js

```diff
@@ -109,19 +109,16 @@ export function reportError(error, opt_associatedElement) {
 
   // Report to console.
   if (self.console) {
+    const output = (console.error || console.log);
     if (error.messageArray) {
-      (console.error || console.log).apply(console,
-          error.messageArray);
+      output.apply(console, error.messageArray);
     } else {
       if (element) {
-        (console.error || console.log).call(console,
-            element.tagName.toLowerCase() +
-                (element.id ? ' with id ' + element.id : '') + ':',
-            error.message);
+        output.call(console, error.message, element);
```

Comment:
Makes sense. Main reason for me is to make this API easier to use -- right now the only way to render the element is to set `error.messageArray`, which isn't done in `error()` or `createError()`.

Before: 
![screen shot 2017-02-01 at 12 12 05 pm](https://cloud.githubusercontent.com/assets/20525523/22517818/6a4b1258-e878-11e6-8e20-06286ce409df.png)

After:
![screen shot 2017-02-01 at 12 16 27 pm](https://cloud.githubusercontent.com/assets/20525523/22517830/759e2910-e878-11e6-88d7-4ea3860be45e.png)

For comparison, error generated by `assert()`:
![screen shot 2017-02-01 at 12 17 01 pm](https://cloud.githubusercontent.com/assets/20525523/22517847/845ac7b0-e878-11e6-98b1-85146f16bcda.png)